### ISSUE TITLE
[mtail] Can't figure out how to make coverage build work

### ISSUE THREAD

**Issue by jaqx0r** [google/oss-fuzz#3293]
Can you help me figure out what's wrong here?

https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=19640 

The build.sh is assuming all the environment variables are passed through to the makefile, which works for the sanitizer, but not coverage, and now I assume there's an environment variable missed but can't figure it out.

**Comment by Dor1s** [comment]
As per the log from https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=19640#c14, it looks like the build itself succeeds (i.e. compilation), but somehow your fuzz target doesn't produce any coverage dumps.

**Comment by Dor1s** [comment]
Ah, this is Go project. Sorry, coverage for Go is not yet supported. We have a workaround that should have prevented coverage failure being reported to you, but it doesn't work because of the reason mentioned in https://github.com/google/oss-fuzz/issues/3297

**TL;DR**: sorry for the noise, we should not have reported that as an error, and we will fix it as soon as we agree on https://github.com/google/oss-fuzz/issues/3297.

**Comment by jaqx0r** [comment]
Great, thanks.  I can't seem to close the bug as wontfix, is that possible?

**Comment by Dor1s** [comment]
I can WontFix it but I'm not sure it won't come back again until we fix #3297. Good news is that we'll address it soon, and https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=19640 will get auto-closed.

**Comment by jaqx0r** [comment]
Thanks!
